composer.json

```diff
@@ -22,7 +22,7 @@
         "php": ">=7.1",
         "composer/installers": "^1.4",
         "vlucas/phpdotenv": "^2.0.1",
-        "johnpbloch/wordpress": "5.2.4",
+        "johnpbloch/wordpress": "5.3.2",
```

Comment:
Hi Stephanie!!! :D 
Thank you. Would you kindly replace the version number with a * and run composer update again?